src/documents/PromisesAPI.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Promises
+layout: default
+category: Infusion
+---
+
+# Infusion Promises API
```

Comment:
The Markdown files shouldn't have a level one header explicitly. The text in the "title" attribute above will be used as the level one header for the page in the rendered HTML.